### PR TITLE
Add header icons

### PR DESCRIPTION
**Pull request by scottaubrey**

### CODE REVIEW

File: src/article/article.ts [hunk 1]
```diff
@@ -103,21 +103,45 @@ const getHeader = (articleDom: DocumentFragment): string => {
...
-  const identifiers = articleDom.querySelector('.content-header__identifiers');
+  const dois = getDois(articleDom);
 
   return `<div class="content-header">
     ${headline?.outerHTML}
     ${authors?.outerHTML}
     ${affiliations?.outerHTML}
-    ${identifiers?.outerHTML}
+    <div class="content-header__footer">
+      ${dois}
```
Comment (discodavey) [inline src/article/article.ts:117]: There would only one doi per page so should we rename this to doi?

File: src/article/article.ts [hunk 2]
```diff
@@ -103,21 +103,45 @@ const getHeader = (articleDom: DocumentFragment): string => {
...
+      <ul class="content-header__icons">
+        <li>
+          <a href="https://en.wikipedia.org/wiki/Open_access" class="content-header__icon content-header__icon--oa">
+                <span class="visuallyhidden">Open access</span>
```
Comment (discodavey) [inline src/article/article.ts:121]: Indentation is incorrect

File: src/article/styles/header.scss [hunk 3]
```diff
@@ -71,6 +71,69 @@
...
+.content-header__identifier a {
+  @include typography.identifiers-list-text();
+  text-decoration: none;
+}
+
+.identifier__name {
+  display: none;
+}
+
+
```
Comment (discodavey) [inline src/article/styles/header.scss:100]: Remove blank line

File: src/article/styles/header.scss [hunk 4]
```diff
@@ -71,6 +71,69 @@
...
+  height: 26px;
+  width: 16px;
+}
+.content-header__icon {
+  vertical-align: middle;
+  background-repeat: no-repeat;
+  background-position: left bottom;
+  display: inline-block;
+  background-size: contain;
+}
```
Comment (discodavey) [inline src/article/styles/header.scss:120]: reorder to being alphabetical

File: src/article/article.ts (hunk 1 above)
Comment (scottaubrey) [reply src/article/article.ts]: Ah, sure. I didn't know to expect that, and Chris's design has "show all" in it. I've changed to expect (and test) for a single DOI, and make sure we return just one, but still as a list for now, until I figure out what show all is for.

File: src/article/styles/header.scss [hunk 5]
```diff
@@ -71,6 +71,69 @@
...
+  clip: rect(0 0 0 0);
+  height: 1px;
+  margin: -1px;
+  overflow: hidden;
+  padding: 0;
+  position: absolute;
+  width: 1px;
+  -webkit-user-select: none;
+  -moz-user-select: none;
+  -ms-user-select: none;
```
Comment (discodavey) [inline src/article/styles/header.scss:137]: Different browser selectors are not needed as all browsers support none.

**Review by discodavey — APPROVED** [review]
All ok.

**Pull request merged**